js/web/lib/onnxjs/backends/webgl/ops/instance-normalization.ts

```diff
@@ -12,8 +12,9 @@ export class WebGLInstanceNormalization extends InstanceNormalization {
     if (!this.artifacts) {
       this.artifacts = [];
       const programInfos = this.createProgramInfos(inferenceHandler, inputs);
-      programInfos.forEach((pi) => {
-        const artifact = inferenceHandler.session.programManager.build(pi);
+      programInfos.forEach((programInfo) => {
+        const artifact = inferenceHandler.session.programManager.build(
+            programInfo, programInfo.name === undefined ? '' : programInfo.name);
```

Comment:
you can use [?? operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to shorten this statement.